app/components/channel_item/channel_item.tsx

```diff
@@ -123,7 +123,7 @@ const ChannelListItem = ({
     const styles = getStyleSheet(theme);
 
     // Make it brighter if it's not muted, and highlighted or has unreads
-    const isBright = isUnread || mentionsCount > 0;
+    const isBright = (!isMuted && isUnread) || mentionsCount > 0;
```

Comment:
@enahum to clarify: 

- We should only bold the channel name if there are mentions in the muted channel. 
- Muted channels should always display the text and icon as muted (reduced opacity) regardless of if it has unreads or mentions.

Reply:
Oh ok, I'll make that change to and ensure that is the case. Thanks for clarifying

Reply:
@esethna done